### PR TITLE
Release 1.5.0

### PR DESCRIPTION
**Pull request by cadmiumcat**
- Renames dp-search-query to dp-search-api
- Updates Go version and license
- Updates audit

### CODE REVIEW

File: ci/build.yml [hunk 1]
```diff
@@ -6,10 +6,10 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.12.12
+    tag: 1.15.7
```
Comment (nshumoogum) [inline ci/build.yml:9]: This can be bumped to 1.15.8

File: ci/unit.yml [hunk 2]
```diff
@@ -6,13 +6,13 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.12.12
+    tag: 1.15.7
```
Comment (nshumoogum) [inline ci/unit.yml:9]: Again we can bump this to 1.15.8

**Review by nshumoogum — CHANGES_REQUESTED** [review]

**Review by nshumoogum — APPROVED** [review]

File: ci/build.yml (hunk 1 above)
Comment (cadmiumcat) [reply ci/build.yml:9]: There's another PR to update this 👍

File: ci/unit.yml (hunk 2 above)
Comment (cadmiumcat) [reply ci/unit.yml:9]: There's another PR to update this 👍

**Pull request merged**